packages/ChangesReloaded-Core.package/CRBrowser.class/instance/switchToPowerfulView.st

```diff
@@ -0,0 +1,5 @@
+button-actions
+switchToPowerfulView
```

Comment:
```suggestion
switchToDetailedView
```

Reply:
I hope it works like this... like I change the method name within the file that is named after the old name... I just hope github can handle that xD

Reply:
Oh, I'm not sure Squeak can handle that correctly or if it always expects method names to match the name of the file they're in – we can try though. The cleanest solution would probably be to just refactor the name in the image (and it's easier too because we don't need to change the references as well).